src/styles/shared/components/Buttons/_index.scss

```diff
@@ -0,0 +1,168 @@
+@use "sass:map";
+@use "@hypothesis/frontend-shared/styles/mixins/focus";
+@use './_mixins';
+@use "../../variables" as var;
+
+// Design tokens from variables
+$-settings: (
+  'border-radius': var.$border-radius,
+  // Internal margin between elements, e.g. SVG and label
+  'margin': 0.5em,
+  'touch-target-size': var.$touch-target-size,
+  'color-g1': var.$color-grey-1,
+  'color-g2': var.$color-grey-2,
+  'color-g3': var.$color-grey-3,
+  'color-gsemi': var.$color-grey-semi,
+  'color-gmid': var.$color-grey-mid,
+  'color-g6': var.$color-grey-6,
+  'color-g7': var.$color-grey-7,
+  'color-brand': var.$color-brand,
+  'color-link-hover': var.$color-link-hover,
+);
+
+// Configure this module to override any of the default settings values
+@mixin configure($settings: null) {
+  @if $settings {
+    $-settings: map.merge($-settings, $settings) !global;
```

Comment:
Does `map` work by only overriding settings that are provided or does a custom button class using this mixin need to pass in all new values?

Reply:
`map.merge` is pretty much analogous to `Object.assign` or object spread. It extends the map; no need to provide values for all entries.